<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Person"/>
    <addaction name="actionAdd_Organizatie"/>
    <addaction name="actionAdd_Competitie"/>
    <addaction name="actionAdd_Categorie_Varsta"/>
    <addaction name="actionAdd_Categorie_Greutate"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuTables">
    <property name="title">
     <string>Tables</string>
    </property>
    <addaction name="actionSportivi"/>
    <addaction name="actionOrganizatie"/>
    <addaction name="actionCompetitie"/>
    <addaction name="actionVarsta"/>
    <addaction name="actionGreutate"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuTables"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Person">
   <property name="text">
    <string>Add Person</string>
   </property>
  </action>
  <action name="actionAdd_Organizatie">
   <property name="text">
    <string>Add Organizatie</string>
   </property>
  </action>
  <action name="actionAdd_Competitie">
   <property name="text">
    <string>Add Competitie</string>
   </property>
  </action>
  <action name="actionAdd_Categorie_Varsta">
   <property name="text">
    <string>Add Categorie Varsta</string>
   </property>
  </action>
  <action name="actionAdd_Categorie_Greutate">
   <property name="text">
    <string>Add Categorie Greutate</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionSportivi">
   <property name="text">
    <string>Sportivi</string>
   </property>
  </action>
  <action name="actionOrganizatie">
   <property name="text">
    <string>Organizatie</string>
   </property>
  </action>
  <action name="actionCompetitie">
   <property name="text">
    <string>Competitie</string>
   </property>
  </action>
  <action name="actionVarsta">
   <property name="text">
    <string>Varsta</string>
   </property>
  </action>
  <action name="actionGreutate">
   <property name="text">
    <string>Greutate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
